sound/soc/sof/intel/hda.h

```diff
@@ -471,6 +471,7 @@ struct sof_intel_hda_stream {
 	struct hdac_ext_stream hda_stream;
 	struct sof_intel_stream stream;
 	int host_reserved; /* reserve host DMA channel */
+	bool d0i3_compatible;
```

Comment:
how about name this to sth like dmi_l1_capable and then we set it true for DMA trace stream also?

Reply:
@ranj063 Hmm, Keyon has a point. To describe DMA stream properties, capability or compatibility with opportunistic DMI L1 entry seems more fitting than the more abstract "D0i3x-compatibility" which is more a property of the whole logical stream (all resources needed for the stream need to fit to the low-power memory/hw).

So here dmi_l1_capable (or "dmi_l1_compatible") would fit better. As this is Intel specific, upper layers should not use this term, so only in hda.h/hda-stream.c/hda-pcm.c.

Reply:
it's more 'dmi_l1_required', no? we can enable L1 on all streams, be they D0i3 compatible or not. We chose not to do so in most cases, but that WoV capture is an exception coming from a hardware requirement.

Reply:
@plbossart I think dmi_l1_compatible is a better choice. "required" would need to be backed up with a confirmation that it was honored.

Reply:
ok, let's leave it as is then.

Reply:
@kv2019i @plbossart @keyonjie now updated to include trace DMA as well.